wrestling_scoreboard_client/lib/view/screens/more/settings/settings.dart

```diff
@@ -392,28 +392,31 @@ class _FontSelectionDialog extends StatelessWidget {
   Widget build(BuildContext context) {
     final localizations = AppLocalizations.of(context)!;
 
-    final List<MapEntry<String?, String>> fontFamilies = [MapEntry<String?, String>(null, localizations.systemSetting)];
-    fontFamilies.addAll(GoogleFonts.asMap().keys.map((String e) => MapEntry<String?, String>(e, e)).toList());
+    final List<String?> fontFamilies = [null];
+    fontFamilies.addAll(GoogleFonts.asMap().keys.toList());
     return RadioDialog<String?>(
       shrinkWrap: false,
       itemCount: fontFamilies.length,
       builder: (index) {
         final fontFamily = fontFamilies[index];
+        final fontStyle = fontFamily != null
```

Comment:
That is a good approach to handle the style in advance. Was about to propse a list of tuples `List<(String fontFamily, FontStyle Function() callStyle)>` so that the list can be predefined, but this works just as well.